.husky/pre-commit

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env sh
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
For whom is unfamiliar with bash, `[ -s FILE ]` evaluate  True if file exists and is not empty.